src/module/actor/sheet/base.js

```diff
@@ -1010,58 +1029,39 @@ export class ActorSheetSFRPG extends ActorSheet {
                 ui.notifications.warn(game.i18n.format("SFRPG.ActorSheet.Inventory.Interface.ItemCollectionPickupNoGMError"));
             }
             return;
-        } else if (parsedDragData.pack) {
```

Comment:
We can't completely rip these lines out yet. They might have to stay in until we've completely updated the system for v10 compatibility.

Reply:
Added version checks to these to ensure they function on v9.